safe_control_gym/hyperparameters/hpo.py

```diff
@@ -0,0 +1,306 @@
+""" The implementation of HPO class
+
+Reference:
+    * stable baselines3 https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/rl_zoo3/hyperparams_opt.py
+    * Optuna: https://optuna.org
+
+Reruirement:
```

Comment:
Requirement. Also, this is installed by the pyproject.toml rt so no need to mention